test/client/fixtures/sandbox/code-instrumentation/location-test.js

```diff
@@ -10,6 +10,27 @@ var Promise       = hammerhead.Promise;
 var nativeMethods = hammerhead.nativeMethods;
 var browserUtils  = hammerhead.utils.browser;
 var domUtils      = hammerhead.utils.dom;
+var navigateTo    = hammerhead.navigateTo;
+
+const SHOULD_ADD_TRAILING_SLASH = [
+    'http://example.com',
+    'https://example.com',
+    'http://localhost',
+    'https://localhost',
+    'http://localhost:8080',
+    'https://localhost:8080',
+    'http://127.0.0.1',
+    'https://127.0.0.1',
+    'http://127.0.0.1:8080',
+    'https://127.0.0.1:8080'
+];
```

Comment:
Let's reduce testcases and combine them to the single contant:
```js
var testcases = [
     {
       url: 'http://example.com'     /*protocol + host*/
       shoudAddTrailingSlash: true
      }, 
      {
         url: https://localhost:8080 /*protocol + host + port*/,
         shoudAddTrailingSlash: true
      },
      {  
         url: about:blank  /*special page*/,
         shoudAddTrailingSlash: false
      }, 
      {
       url: http://example.com/page.html, /*url with path*/
         shoudAddTrailingSlash: false      
      },
     linux file protocol url,
    window file protocol url
];
```

Reply:
Done.